irr/src/COGLESExtensionHandler.cpp

```diff
@@ -12,7 +12,7 @@
 #include "SMaterial.h"
 #include "fast_atof.h"
 
-#if defined(_IRR_COMPILE_WITH_ANDROID_DEVICE_) || defined(_IRR_COMPILE_WITH_WINDOWS_DEVICE_)
+#if defined(_IRR_COMPILE_WITH_WINDOWS_DEVICE_)
```

Comment:
i think this should rather check for `__ANDROID__`
The NDK has both headers but I think the EGL one is more correct:
<pre>
ndk/26.2.11394342/toolchains/llvm/prebuilt/linux-x86_64/sysroot/usr/include/GLES/egl.h
ndk/26.2.11394342/toolchains/llvm/prebuilt/linux-x86_64/sysroot/usr/include/EGL/egl.h
</pre>

Reply:
`COGLESExtensionHandler.cpp` is only used by the gles1 driver, which doesn't work with SDL, so it doesn't work anymore on Android.

Reply:
right. Android and Linux were our only target platforms for this driver so it can actually be deleted afterwards.